farm/modeling/prediction_head.py

```diff
@@ -947,6 +948,10 @@ def __init__(self, layer_dims=[768,2],
                                   This is decoupled from n_best on document level, since predictions on passage level are very similar.
                                   It should have a low value
         :type n_best_per_sample: int
+        :param filter_range: The maximum distance that the start indices or end indices of two answers need to have to be handled as duplicates.
```

Comment:
Maybe simplify naming and docstring?

Name the parameter: "duplicate_filtering" and the description "Answers are filtered based on their position. Both start and end position of the answers are considered. The higher the value answers that are more apart are filtered out."
Then give the exmaples.